### ISSUE TITLE
consistent use of ods:AgentClass

### ISSUE THREAD

**Issue by DavidFichtmueller** [DiSSCo/openDS#150]
### Suggestion

The use of the ods:Agent class seems a bit inconsistent. Many classes have objectProperties pointing to it, but there are still many properties that just offer a single non repeatable text field for an agent name and a separate non repeatable text field for an agent it. These elements could be made more consistent, by replacing them with an object property to the agent class. This would also add the proper connection between the names and identifiers, in case of multiple agents. 
These are the items we have identified as suitable for a replacement: 
- ods:CronometricAge - chrono:chronometricAgeDeterminedBy
- ods:DigitalSpecimen - dwc:recordedBy 
  - (there is already a hasAgent property, which could contain this information)
- ods:DigitalSpecimen - dwc:recordedByID
  - as above
- ods:EntityRelationship - dwc:relationshipAccordingTo 
  - (in this case there is already the property ods:RelationshipAccordingToAgent, so this would contain duplicate information anyways)
- ods:Event - ods:collectorName 
  - (even if this will not be replaced by an hasAgent property, why was a dedicated term defined and not just dwc:recordedBy reused, as it is pretty much the same thing. 
- ods:Event - ods:collectorID 
  - (even if this will not be replaced by an hasAgent property, why was a dedicated term defined and not just dwc:recordedByID reused, as it is pretty much the same thing.
- ods:GeoReference - dwc:georeferencedBy
- ods:Identification - dwc:identifiedBy
  - this is of particular importance as it would allow for multiple agents with different roles to be connected to the identification, which is useful to assign the initial identification and the later verification to different agents. 
- ods:Identification - dwc:identifiedByID
  - as above
- ods:MaterialEntity- dwc:recordedBy 
  - (there is already a hasAgent property, which could contain this information)
- ods:MaterialEntity- dwc:recordedByID
  - as above

**Comment by samleeflang** [comment]
Yes, generally I agree with this idea and it has crossed my mind several times.
However, the `dwc` terms are widely recognised and used.
We could pull apart these names and put them in the agent (and we probably will do this anyway) but keeping it in the original dwc might significantly help with backwards compatibility/findability. 
On the other hand, the vagueness of a `dwc:recordedBy` (is it the collector or the one the recorded/digitiser) and the issue with multiple agents makes me sway towards removing these fields and use agents only.
In the end it didn't hurt to keep these fields in (and duplicate some data), but it is a good point, and maybe we should reconsider.
The new `ods` fields for collector I will remove. 
I will discuss with the team whether we see a major issue in removing these fields.
